src/lib.rs

```diff
@@ -63,6 +70,24 @@
 //!
 //! assert!(result.is_ok());
 //! ```
+//!
+//! To deal with fatal errors, return `OperationResult` directlythread::spawn(move || {
```

Comment:
Looks like a copy/paste error at the end of the line here.

Reply:
arg, vscode doesn't discriminate between prose and code well :/.